upstairs/src/client.rs

```diff
@@ -2692,6 +2723,24 @@ impl ClientIoTask {
             + std::marker::Send
             + 'static,
     {
+        // Delay communication with this client based on backpressure, to keep
+        // the three clients relatively in sync with each other.
+        //
+        // We don't need to delay writes, because they're already constrained by
+        // the global backpressure system and cannot build up an unbounded
+        // queue.  This is admittedly quite subtle; see crucible#1167 for
+        // discussions and graphs.
+        if !matches!(
```

Comment:
reads and flushes, those we should respect the backpressure delay for.

What about `WriteUnwritten`, and/or any of the repair operations?
If we block those here, will that have any impact elsewhere in the system?
There should never be more than one `WriteUnwritten`, or four of the
repair jobs outstanding at a time, so I don't really think there could be
too much of an impact if we slow them down.

Edit: It looks like we don't set timing data for repair operations, but they
will still be subject to delays if reads have backed up enough such that
we have set a delay.

Reply:
`WriteUnwritten` don't go through the normal backpressure system, so I think they qualify for delays here (but am not totally sure).

I'm also not too worried about it, since the max per-client delay is 10ms, and we only hit that if one client is **a full second** behind the others.  The intent for this delay is to serve as a small nudge, not "blocking" in any significant sense.